Std/Data/Array/Attach.lean

```diff
@@ -0,0 +1,16 @@
+import Std.Data.List.Basic
```

Comment:
Can this go in `Std.Data.Array.Basic`? Assuming it doesn't increase the imports.